apps/cricket-score-frontend/README.md

```
@@ -154,23 +156,25 @@
 
-1. **Start the development server**
+1. **Start the backend (optional if using hosted API)**
    ```bash
-   pnpm dev
+   pnpm backend:dev
    ```
 
-2. **Open your browser**
-   Navigate to [http://localhost:3000](http://localhost:3000) to see the application.
+2. **Start the frontend development server**
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)